client/my-sites/sidebar-unified/fallback-data.js

```diff
@@ -297,31 +304,35 @@ export default function buildFallbackResponse( { siteDomain = '' } = {} ) {
 					type: 'submenu-item',
 					url: `/customize/${ siteDomain }`,
 				},
-				...( shouldShowApperanceHeaderAndBackground && [
-					{
-						parent: 'themes.php',
-						slug: 'themes-header',
-						title: translate( 'Header' ),
-						type: 'submenu-item',
-						url: `https://${ siteDomain }/wp-admin/customize.php?return=%2Fwp-admin%2F&autofocus%5Bcontrol%5D=header_image`,
-					},
-					{
-						parent: 'themes.php',
-						slug: 'themes-background',
-						title: translate( 'Background' ),
-						type: 'submenu-item',
-						url: `https://${ siteDomain }/wp-admin/customize.php?return=%2Fwp-admin%2F&autofocus%5Bcontrol%5D=background_image`,
-					},
-				] ),
-				...( showShowThemeOptions && [
-					{
-						parent: 'themes.php',
-						slug: 'themes-options',
-						title: translate( 'Theme Options' ),
-						type: 'submenu-item',
-						url: `https://${ siteDomain }/wp-admin/`,
-					},
-				] ),
+				...( shouldShowApperanceHeaderAndBackground
+					? [
+							{
+								parent: 'themes.php',
+								slug: 'themes-header',
+								title: translate( 'Header' ),
+								type: 'submenu-item',
+								url: `https://${ siteDomain }/wp-admin/customize.php?return=%2Fwp-admin%2F&autofocus%5Bcontrol%5D=header_image`,
+							},
+							{
+								parent: 'themes.php',
+								slug: 'themes-background',
+								title: translate( 'Background' ),
+								type: 'submenu-item',
+								url: `https://${ siteDomain }/wp-admin/customize.php?return=%2Fwp-admin%2F&autofocus%5Bcontrol%5D=background_image`,
+							},
```

Comment:
Themes can add support for one and not for the other, so these might have to be split

Reply:
Did a little research, I found:
* On wpcom the backend checks `current_theme_supports( 'custom-header' )` and `( current_theme_supports( 'custom-background' )` separately.
* For an example of it, try using theme "Dara"
* I got theme-related information in calypso to load once, but I'm not sure how.  I think the selector would be something like this:

```js
	import { getActiveTheme, getCanonicalTheme } from 'calypso/state/themes/selectors';
	const theme = useSelector( ( state ) => {
		const currentThemeId = getActiveTheme( state, selectedSiteId );
		return {
			currentThemeId,
			currentTheme: getCanonicalTheme( state, selectedSiteId, currentThemeId ),
		};
	} );
```

The only part I was able to verify was `currentThemeId`, I wasn't able to see what `getCanonicalTheme` requested.

Reply:
We probably need to actively dispatch an action to get the Theme data. Let me try that.

Reply:
I've removed the need for these tests. I feel it's getting overly complicated for the ROI.